lib/concurrent/atomic/count_down_latch.rb

```diff
@@ -55,7 +55,7 @@ module Concurrent
   # @!macro internal_implementation_note
   CountDownLatchImplementation = case
                                  when Concurrent.on_jruby?
-                                   JavaCountDownLatch
```

Comment:
hey, could we get some details on why the 'native' JRuby stuff, seems to be going away?
understand the limited time for maintenance but still seems weird these had to go ...

Reply:
Sure, please have a look at #717. I am updating the changelog before release, and when the 1.0.6.pre goes out it will mention this as well.

Reply:
Please ask any follow up questions there.